code-challenges/401/linkedList/__tests__/linked-list.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const modules = require('../linked-list.js');
+const LL = modules.LL;
```

Comment:
You don't have a Node class in your linked list, currently you are only requiring in your Linked List class which would have to be referenced with the class name it was being exported with.